pages/03-core/simple-zooming.page.tsx

```diff
@@ -216,7 +216,7 @@ function Charts() {
     });
   const onScatterClearHighlight = () =>
     withLock(() => {
```

Comment:
Let's remove the custom lock mechanism to prove that this solution actually works.